plugins/osdn/subnets.go

```diff
@@ -123,21 +128,23 @@ func (oc *OvsController) SubnetStartNode(mtu uint) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	subnets := result.([]api.Subnet)
+	subnets := result.([]osapi.HostSubnet)
 	for _, s := range subnets {
-		oc.flowController.AddOFRules(s.NodeIP, s.SubnetCIDR, oc.localIP)
+		if s.HostIP != oc.localIP {
+			oc.pluginHooks.AddHostSubnetRules(&s)
```

Comment:
Do we want to return error from AddHostSubnetRules()?  watchSubnets() can log the error and SubnetStartNode() can propagate the error to the higher layers?